test/integration_test/migration_test.go

```diff
@@ -0,0 +1,63 @@
+// +build integrationtest
+
+package integrationtest
+
+import (
+	"testing"
+
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+)
+
+func testBasicMigration(t *testing.T) {
+	logrus.Info("Basic sanity tests for cloud migration")
+	t.Run("sanityMigrationTest", sanityMigrationTest)
+}
+
+func sanityMigrationTest(t *testing.T) {
+	var err error
+	err = dumpRemoteKubeConfig(remoteConfig)
+	require.NoError(t, err, "Error writing to clusterpair.yml")
+
+	info, err := volumeDriver.GetClusterPairingInfo()
+	require.NoError(t, err, "Error writing to clusterpair.yml")
+
+	err = createClusterPair(info)
+	require.NoError(t, err, "Error creating cluster Spec")
+
+	err = schedulerDriver.RescanSpecs(specDir)
+	require.NoError(t, err, "Unable to parse spec dir")
+
+	_, err = schedulerDriver.Schedule("clusterpair",
+		scheduler.ScheduleOptions{AppKeys: []string{"cluster-pair"}})
+	require.NoError(t, err, "Error applying clusterpair")
+
+	logrus.Info("Validated Cluster Pair Specs")
```

Comment:
Put the steps till here in a separate function so different test can call it to set up the cluster pair.